packages/gatsby-theme-newrelic/src/i18n/translations/jp.json

```diff
@@ -50,7 +50,7 @@
     "ukSlaveryAct": "英国現代奴隷法"
   },
   "searchInput": {
-    "placeholder": "Docs.newrelic.com、Developer.newrelic.com、OpenSource.newrelic.comを検索"
+    "placeholder": "を検索"
```

Comment:
I assume that this is the same as `search docs` in japanese?

Reply:
This is just `Search` for now! Julien will give me the translations once he has them. Both kr and jp keep their former `Search`. 👌🏻